MyApp/app/src/main/java/com/example/janari/myapp/EnterIncomeAndExpenses.java

```diff
@@ -0,0 +1,53 @@
+package com.example.janari.myapp;
```

Comment:
`com.example.janari.myapp`... Who is Janari and why is the name of a application "MyApp"? I reccomend researching about project naming conventions.